app/js/reducers/settingEncounterRoleReducer.js

```diff
@@ -17,8 +17,6 @@ const settingEncounterRoleReducer = (
       return { ...state, isLoading: action.status };
     case SETTING_ENCOUNTER_ROLE_FAILURE:
       return { ...state, roleError: action.error };
-    case NETWORK_ERROR:
```

Comment:
Do you mind explaining a bit how removing the network error lines is related to the work for this ticket?

Reply:
Actually, it doesn't directly have anything to with this ticket but having to refactor the test for this file made me realise that this line was not been used anywhere and therefore unnecessary, hence the need to remove it.